app/helpers/pipeline_outputs_helper.rb

```diff
@@ -1,12 +1,17 @@
 module PipelineOutputsHelper
   Client = Aws::S3::Client.new
 
-  def get_taxid_fasta(pipeline_output, taxid)
-    uri_parts = pipeline_output.sample.sorted_taxid_annotated_fasta_s3_path.split("/", 4)
+  def get_taxid_fasta(pipeline_output, taxid, tax_level, hit_type)
+    uri = pipeline_output.sample.s3_paths_for_taxon_byteranges[tax_level][hit_type]
```

Comment:
Maybe add some documentation of what the uri looks like?

Reply:
ok